### ISSUE TITLE
Light theme improvements needed

### ISSUE THREAD

**Issue by Cassolotl** [tootsuite/mastodon#7566]
There are some parts of the new light theme that are too low contrast, so I'll post them here.

Tagging @Sylvhem, as requested. :)

~

Links and X not light enough in reply UI:

![screen shot 2018-05-21 at 12 59 48](https://user-images.githubusercontent.com/6607512/40306575-17896912-5cf7-11e8-908e-d360080430b3.png)

~

Links (tags, hashtags, URLs) are not a different enough colour compared to the rest of the text:

![screen shot 2018-05-21 at 13 01 45](https://user-images.githubusercontent.com/6607512/40306602-30e75144-5cf7-11e8-8519-4673435593cf.png)

~

When you like something, the star spins but doesn't change colour - you have to reload the page to see it turn yellow:

![like](https://user-images.githubusercontent.com/6607512/40306653-6b5c64c2-5cf7-11e8-8b98-2ba204a2cbbc.gif)

~

Unselected titles and icons in the privacy dropdown are too light:

![screen shot 2018-05-21 at 13 04 03](https://user-images.githubusercontent.com/6607512/40306676-80b8aa24-5cf7-11e8-8c8b-1abb51b2186b.png)

~

As and if/when I find more I'll add them in new posts to this thread. :)

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [ ] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by Cassolotl** [comment]
Not enough contrast in the toot text here:

![photo_2018-05-22_09-37-25](https://user-images.githubusercontent.com/6607512/40351206-c0b6b83a-5da3-11e8-9c93-077cbe14510f.jpg)

Not enough contrast between the *backgrounds* of DMs and normal toots:

![screen_shot_2018-05-22_at_09_37_00](https://user-images.githubusercontent.com/6607512/40351258-e0eef176-5da3-11e8-9b2e-2b3753f9d276.png)

**Comment by Cassolotl** [comment]
Not enough contrast on the scrollbars.

![screen shot 2018-05-26 at 20 38 55](https://user-images.githubusercontent.com/6607512/40579669-d9bafbe6-6124-11e8-8ffd-016880ebbe94.png)

**Comment by Cassolotl** [comment]
Here's a toot page:

![screen shot 2018-05-27 at 18 25 58](https://user-images.githubusercontent.com/6607512/40588828-7f61dff4-61db-11e8-981a-427dbdf2d83a.png)

It's unreadable except for the hashtags. The hashtags are blue here, and I feel like link things (hashtags, @, URLs) should be blue in the web UI too!

**Comment by Cassolotl** [comment]
Not enough contrast on the little arrow - should it be the same colour as the toot background?

![screen shot 2018-05-28 at 11 31 13](https://user-images.githubusercontent.com/6607512/40610563-ce54eee6-626a-11e8-9d89-998d67f05a58.png)

**Comment by Gargron** [comment]
>should it be the same colour as the toot background?

Yes, definitely

**Comment by ChameleonScales** [comment]
Personally I find that the light theme could use some more saturation to pump it up.

**Comment by joachimesque** [comment]
Didn't want to add a new issue, so here's my improvement proposal: better legibility for links.

Currently, links are not underlined, and not highly differentiated from regular text. It's a usability/accessibility problem. My proposal would be to change the color of the links.

Before:
![screenshot_2018-08-23 boitam eu 1](https://user-images.githubusercontent.com/1781070/44521619-f693ed00-a6d3-11e8-827b-a84b79e48f91.png)

After:
![screenshot_2018-08-23 boitam eu](https://user-images.githubusercontent.com/1781070/44521625-fe539180-a6d3-11e8-9ee6-7f484ad76992.png)

Another solution would be to add an underline.

**Comment by Cassolotl** [comment]
@joachimesque I like this a lot! The links being indistinguishable from normal text makes the light theme unusable for me at the moment.

**Comment by JaredLethal** [comment]
What about using mastodon's blue ?

**Comment by joachimesque** [comment]
Good idea @JaredLethal, in my screen capture I used the "Toot" button blue, but I don't know if it's the official Mastodon's blue.
